test/jetstream_test.go

```diff
@@ -82,6 +83,7 @@ func RunBasicJetStreamServer() *server.Server {
 	opts := DefaultTestOptions
 	opts.Port = -1
 	opts.JetStream = true
+	opts.NoLog = true
```

Comment:
This should be part of DefaultTestOptions no?

Reply:
yeah not sure how that crept in there, removing

Reply:
Actually, for websocket/mqtt tests, I do set this intentionally to `NoLog = false` and set a dummy logger with debug and trace enabled:
```
    opts.NoLog = false
    l := &DummyLogger{}
    s.SetLogger(l, true, true)
```
The reason? This exercises all code that has debug/trace, making sure that we don't have situations where we may be tracing something that causes a dereference of a nil pointer. As an added bonus, it helps with code coverage ;-)

Reply:
Might be a good idea.